docs/cookbook/extending-product-list.md

```diff
@@ -66,9 +66,123 @@ class ListedProductView extends BaseListedProductView
 }
 ```
 
-### 2. Extend `ListedProductViewFactory` so it returns the new required data
+### 2. Add new attribute to ElasticSearch
+
+In order to add new attribute to ElasticSearch you need to add it to the structure first.
+You can do that by adding it to `mappings` in all `src/Shopsys/ShopBundle/Resources/definition/product/*.json` files like this:
+```json
+// ...
+  "mappings": {
+    "_doc": {
+      "properties": {
+        "brand_name": {
+          "type": "text"
+        },
+        
+        // ...
+      }
+    }
+  }
+// ...
+```
+
+### 3. Export new attribute to ElasticSearch
+
+The class responsible for exporting products to ElasticSearch is `ProductSearchExportWithFilterRepository` so we need to extend it and add the attribute to method `getProductsData`.
```

Comment:
When it's already targeted to the next major I'm thinking about naming the class as `ProductSearchExport` or `ProductElasticExport`.
What do you think? And should it be done in this PR? #questioneverything

Reply:
I don't think that this should be done in this PR